linalg/sparsemat.hpp

```diff
@@ -346,30 +346,45 @@ class SparseMatrix : public AbstractSparseMatrix
                          const double a = 1.0) const;
 
    /** @brief Build and store internally the transpose of this matrix which will
-       be used in the methods AddMultTranspose() and MultTranspose(). */
+       be used in the methods AddMultTranspose(), MultTranspose(), and
+       AbsMultTranspose(). */
    /** If this method has been called, the internal transpose matrix will be
        used to perform the action of the transpose matrix in AddMultTranspose(),
-       and MultTranspose().
+       MultTranspose(), and AbsMultTranspose().
 
        Warning: any changes in this matrix will invalidate the internal
        transpose. To rebuild the transpose, call ResetTranspose() followed by a
        call to this method. If the internal transpose is already built, this
        method has no effect.
 
-       When any non-default backend is enabled, i.e. Device::IsEnabled() is
-       true, the methods AddMultTranspose(), and MultTranspose(), require the
-       internal transpose to be built. If that is not the case (i.e. the
-       internal transpose is not built), these methods will raise an error with
-       an appropriate message pointing to this method. When using the default
-       backend, calling this method is optional.
+       When any non-serial-CPU backend is enabled, i.e. the call
+       `Device::Allows(~Backend::CPU_MASK)` returns true, the above methods
```

Comment:
```suggestion
       `Device::Allows(~%Backend::CPU_MASK)` returns true, the above methods
```
to prevent Doxygen from misinterpreting `~Backend::CPU_MASK`.

Reply:
I found a slightly different solution: add a space after `~` -- that's probably a bit less confusing for someone reading the code and not the doxygen.